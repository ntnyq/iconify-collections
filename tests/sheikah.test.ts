import { createGenerator, presetIcons } from 'unocss'
import { expect, it } from 'vitest'

it('render', async () => {
  const targets = [
    '0',
    '1',
    '2',
    '3',
    '4',
    '5',
    '6',
    '7',
    '8',
    '9',
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
    'h',
    'i',
    'j',
    'k',
    'l',
    'm',
    'n',
    'o',
    'p',
    'q',
    'r',
    's',
    't',
    'u',
    'v',
    'w',
    'x',
    'y',
    'z',
    'exclam',
    'hyphen',
    'period',
    'question',
  ].map(c => `i-sheikah-${c}`)
  const generator = await createGenerator({
    presets: [
      presetIcons({
        collections: {
          sheikah: () => import('../packages/sheikah/icons.json').then(i => i.default as any),
        },
      }),
    ],
  })
  const { css } = await generator.generate(targets.join('\n'))
  expect(css).toMatchSnapshot()
})
