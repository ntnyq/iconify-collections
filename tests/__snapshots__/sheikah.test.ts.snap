// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`sheikah > render 1`] = `
"/* layer: icons */
.i-sheikah-0{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M452.5 452H572v119.5H452.5V452zm-299-179L273 153.5h299V273H273v597.5L153.5 751V273zM751 153.5 870.5 273v478L751 870.5H452.5V751H751V153.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-1{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zM751 751H153.5L273 870.5h478L870.5 751V273L751 153.5V751zM452.5 153.5V452h-299v119.5h299L572 452V153.5H452.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-2{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M273 153.5 153.5 273v179L273 571.5h478L870.5 452V273L751 153.5V452H273V153.5zM870.5 751 751 870.5H273L153.5 751h717zm-418-597.5H572V273H452.5V153.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-3{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M870.5 153.5V273H751V153.5h119.5zM273 751V153.5L153.5 273v478L273 870.5h478L870.5 751H273zm597.5-299H572V153.5H452.5V452L572 571.5h298.5V452z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-4{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 751 273 870.5h179.5L572 751V273L452.5 153.5H273L153.5 273h299v478h-299zM751 153.5 870.5 273v478L751 870.5v-717zm-597.5 418V452H273v119.5H153.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-5{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M572 571.5V452H452.5v119.5H572zm179 299L870.5 751V452H751v299H153.5L273 870.5h478zM870.5 273 751 153.5H273L153.5 273v298.5H273V273h597.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-6{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M870.5 273 751 153.5H572L452.5 273v478L572 870.5h179L870.5 751H572V273h298.5zM273 870.5 153.5 751V273L273 153.5v717zM870.5 452v119.5H751V452h119.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-7{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 870.5V751H273v119.5H153.5zM751 273v597.5L870.5 751V273L751 153.5H273L153.5 273H751zM153.5 571.5h299v299H572v-299L452.5 452h-299v119.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-8{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M751 870.5 870.5 751V571.5L751 452H273L153.5 571.5V751L273 870.5v-299h478v299zM153.5 273 273 153.5h478L870.5 273h-717zM572 870.5H452.5V751H572v119.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-9{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M870.5 870.5H751V751h119.5v119.5zM273 273h597.5L751 153.5H273L153.5 273v478L273 870.5V273zm299 597.5v-299h298.5V452H572L452.5 571.5v299H572z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-a{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5V751h478V273H472v279h199.5v119.5H472L352.5 552V273L472 153.5h279L870.5 273v478L751 870.5H273L153.5 751V273zM552 352.5h119.5V472H552V352.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-b{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zm597.5 0h119.5V273H751V153.5zm-398.5 0H472v199L352.5 472h-199V352.5h199v-199zm199.5 0h119.5v199h199V472h-199L552 352.5v-199zm-279 717L153.5 751v-79.5L273 552h478l119.5 119.5V751L751 870.5v-199H273v199zM352.5 751h319v119.5h-319V751z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-c{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h79.5L472 273v79.5L352.5 472H273v80h79.5L472 671.5V751L352.5 870.5H273L153.5 751h199v-79.5H273L153.5 552v-80L273 352.5h79.5V273h-199zm717 0L751 153.5h-79.5L552 273v79.5L671.5 472H751v80h-79.5L552 671.5V751l119.5 119.5H751L870.5 751h-199v-79.5H751L870.5 552v-80L751 352.5h-79.5V273h199z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-d{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273v478L751 870.5V273H153.5zm0 79.5H552L671.5 472v279L552 870.5H273L153.5 751H552V472H153.5V352.5zm0 199.5H472v119.5H153.5V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-e{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zm597.5 0h119.5V273H751V153.5zM352.5 273 472 153.5h80L671.5 273v199H552V273h-80v199H352.5V273zm-199 79.5H273V552h79.5L472 671.5V751L352.5 870.5H273L153.5 751h199v-79.5H273L153.5 552V352.5zm717 0H751V552h-79.5L552 671.5V751l119.5 119.5H751L870.5 751h-199v-79.5H751L870.5 552V352.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-exclam{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M273 870.5 153.5 751V273L273 153.5h179.5L572 273v478h179V153.5L870.5 273v478L751 870.5H572L452.5 751V273H273v597.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-f{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273v199H751V273H273v199H153.5V273zm0 279H273v199h79.5V472L472 352.5h80L671.5 472v279H751V552h119.5v199L751 870.5h-79.5L552 751V472h-80v279L352.5 870.5H273L153.5 751V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-g{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273v478L751 870.5V273H273v279h79.5v-80L472 352.5h80L671.5 472v279L552 870.5H273L153.5 751H552V472h-80v80L352.5 671.5H273L153.5 552V273z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-h{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zm199 0H472V273H352.5V153.5zm199.5 0h119.5V273H552V153.5zm199 0h119.5V273H751V153.5zm-478 717L153.5 751V472L273 352.5h478L870.5 472v279L751 870.5V472H273v398.5zm79.5 0v-199L472 552h80l119.5 119.5v199H552v-199h-80v199H352.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-hyphen{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zM751 751h119.5v119.5H751V751zM352.5 153.5V273H751v179H273L153.5 571.5V751L273 870.5h398.5V751H273V571.5h478L870.5 452V273L751 153.5H352.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-i{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 797 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M40.25 750.75h119.5v119.5H40.25v-119.5zm597 0h119.5v119.5h-119.5v-119.5zm-597-79.5v-398l119.5-119.5h79.5l119.5 119.5v477.5h79.5v-477.5l119.5-119.5h79.5l119.5 119.5v398h-119.5v-398h-79.5v477.5l-119.5 119.5h-79.5l-119.5-119.5v-477.5h-79.5v398H40.25z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-j{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5v199h79.5V273L472 153.5h279L870.5 273v478L751 870.5V273H472v79.5L352.5 472H273L153.5 352.5V273zM552 352.5h119.5V751L552 870.5H273L153.5 751H552V352.5zM153.5 552H273v119.5H153.5V552zm199 0H472v119.5H352.5V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-k{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M273 870.5 153.5 751V273L273 153.5v717zm478-717L870.5 273v478L751 870.5v-717zm-398.5 0H472V472H352.5V153.5zm0 398.5H472v318.5H352.5V552zm199.5 0h119.5v318.5H552V552zm0-80h119.5V153.5H552V472z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-l{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5V552h79.5V273L472 153.5h279L870.5 273H472v279L352.5 671.5H273L153.5 552V273zM552 352.5h318.5V472H552V352.5zm0 199.5h318.5v119.5H552V552zM153.5 751 273 870.5h478L870.5 751h-717z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-m{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273h-717zm717 478L751 870.5H273L153.5 751h717zm-717-79.5H273V552H153.5v119.5zm199 0H472V472L352.5 352.5h-199V472h199v199.5zm199.5 0V472l119.5-119.5h199V472h-199v199.5H552zM751 552h119.5v119.5H751V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-n{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h79.5L472 273v79.5L352.5 472h-199V352.5h199V273h-199zM552 153.5V552H273L153.5 671.5V751L273 870.5h478L870.5 751V273L751 153.5V751H273v-79.5h279L671.5 552V153.5H552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-o{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M352.5 153.5H472V273H352.5V153.5zm199.5 0h119.5V273H552V153.5zM153.5 552H273v318.5L153.5 751V552zm597.5 0h119.5v199L751 870.5V552zM273 352.5h79.5L472 472v279h80V472l119.5-119.5H751v-199L870.5 273v79.5L751 472h-79.5v279L552 870.5h-80L352.5 751V472H273L153.5 352.5V273L273 153.5v199z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-p{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M352.5 153.5h319V273h-319V153.5zm0 199h319V472h-319V352.5zm0 199.5h319v119.5h-319V552zM273 751h478V153.5L870.5 273v478L751 870.5H273L153.5 751V273L273 153.5V751z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-period{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M870.5 870.5V273L751 153.5H273L153.5 273v478L273 870.5h179.5L572 751V452H452.5v299H273V273h478v597.5h119.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-q{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 153.5H273V273H153.5V153.5zm199 0H472V273H352.5V153.5zm0 398.5H472v318.5H352.5V552zm199.5 0h119.5v318.5H552V552zM273 870.5 153.5 751V472L273 352.5h279v-199h119.5v199L552 472H273v398.5zm478-717L870.5 273v478L751 870.5v-717z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-question{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 797 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M40.25 153.75h119.5v119.5H40.25v-119.5zm8.5 298h119.5v119.5H48.75v-119.5zm-8.5 299 119.5 119.5h179l119.5-119.5v-477.5h298.5l-119.5-119.5h-179l-119.5 119.5v477.5H40.25zm597-298h119.5v119.5h-119.5v-119.5zm0 298h119.5v119.5h-119.5v-119.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-r{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 751H273v119.5H153.5V751zm597.5 0h119.5v119.5H751V751zM153.5 472V273L273 153.5h79.5L472 273v79.5h80V273l119.5-119.5H751L870.5 273v199H751V273h-79.5v79.5L552 472h-80L352.5 352.5V273H273v199H153.5zm0 80h199L472 671.5V751h80v-79.5L671.5 552h199v119.5h-199V751L552 870.5h-80L352.5 751v-79.5h-199V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-s{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M352.5 352.5h319V472h-319V352.5zm0 199.5h319v119.5h-319V552zM472 153.5V273H273v478h199v119.5H273L153.5 751V273L273 153.5h199zm80 0V273h199v478H552v119.5h199L870.5 751V273L751 153.5H552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-t{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M352.5 153.5h319V273h-319V153.5zm518 597.5L751 870.5H273L153.5 751h717zm-717-478L273 153.5V552h79.5v-80L472 352.5h80L671.5 472v80H751V153.5L870.5 273v279L751 671.5h-79.5L552 552v-80h-80v80L352.5 671.5H273L153.5 552V273z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-u{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M870.5 751 751 870.5H273L153.5 751h717zm-518-398.5H472V472H352.5V352.5zm199.5 0h119.5V472H552V352.5zm199 319h119.5V273L751 153.5v518zm-79.5-518V273H273v279h398.5v119.5H273L153.5 552V273L273 153.5h398.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-v{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273v79.5L751 472H472v398.5H352.5V472L472 352.5h279V273H273v597.5L153.5 751V273zM552 552h119.5v318.5H552V552zm199 0h119.5v199L751 870.5V552z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-w{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M352.5 273v279L472 671.5h80L671.5 552V273h199L751 153.5h-79.5L552 273v279h-80V273L352.5 153.5H273L153.5 273h199zm-199 79.5H273V751h478V352.5h119.5V751L751 870.5H273L153.5 751V352.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-x{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 352.5H273v319H153.5v-319zm597.5 0h119.5v319H751v-319zM153.5 273 273 153.5h79.5L472 273v79.5h80V273l119.5-119.5H751L870.5 273h-199v79.5L552 472h-80L352.5 352.5V273h-199zm199 478v-79.5L472 552h80l119.5 119.5V751h199L751 870.5h-79.5L552 751v-79.5h-80V751L352.5 870.5H273L153.5 751h199z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-y{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 751 273 870.5v-518H153.5V751zm199-478v597.5H472V273L352.5 153.5H273L153.5 273h199zM751 153.5 870.5 273v398.5H751v-518zm-199 0h119.5V751h199L751 870.5h-79.5L552 751V153.5z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}
.i-sheikah-z{--un-icon:url(\\"data:image/svg+xml;utf8,%3Csvg viewBox='0 0 1024 1024' width='1em' height='1em' xmlns='http://www.w3.org/2000/svg' %3E%3Cpath fill='currentColor' d='M153.5 273 273 153.5h478L870.5 273v279L751 671.5h-79.5L552 552V352.5h119.5V552H751V273H153.5zM273 751V472h79.5v199.5H472V472L352.5 352.5H273L153.5 472v279L273 870.5h478L870.5 751H273z'/%3E%3C/svg%3E\\");-webkit-mask:var(--un-icon) no-repeat;mask:var(--un-icon) no-repeat;-webkit-mask-size:100% 100%;mask-size:100% 100%;background-color:currentColor;color:inherit;width:1em;height:1em;}"
`;
